source/Octopus.Client/Exceptions/IncludeGlobalCannotBeSetFromFalseToTrueException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Octopus.Client.Exceptions
+{
+    public class IncludeGlobalCannotBeSetFromFalseToTrueException : Exception
```

Comment:
This exception doesn't really feel like it helps the user understand why this is an invalid operation. The name also feels a bit odd.

Maybe something like `public InvalidIncludeGlobalConfigurationException(): base("Cannot include global resources when global resources have already been excluded")